Italian translation

amivaleo opened the pull request:
Hi!
I added the italian translation to your extension.
I hope you will appreciate it. :)

OttoAllmendinger commented on src/locale/it/it.po:
```diff
@@ -0,0 +1,193 @@
+"Report-Msgid-Bugs-To: \n"
+"POT-Creation-Date: 2017-04-19 12:23+0200\n"
+"PO-Revision-Date: 2017-04-20 09:43+0200\n"
+"Last-Translator: Raphael Rochet <raphael@rri.fr>\n"
```
Feel free to add your name here :)

amivaleo replied:
Done. :)

OttoAllmendinger commented on src/locale/it/it.po:
```diff
@@ -0,0 +1,193 @@
+"PO-Revision-Date: 2017-04-20 09:43+0200\n"
+"Last-Translator: Raphael Rochet <raphael@rri.fr>\n"
+"Language-Team: \n"
+"Language: fr\n"
```
`Language: it`

amivaleo replied:
Fixed.

OttoAllmendinger commented on src/locale/it/it.po:
```diff
@@ -0,0 +1,193 @@
+# FIXME this is translation of "Default Click Action"
+#: src/prefs.js:107
+msgid "Primary Button"
+msgstr "Comportamento al click"
```
I think this is still the italian translation of "Default click action" or "Click action". Can you change it to "Primary Button"?

amivaleo replied:
Ok, I literally translated that now.
Primary Button -> Pulsante primario

OttoAllmendinger commented on src/locale/it/it.po:
```diff
@@ -0,0 +1,193 @@
+
+#: src/filename.js:27
+msgid "Screenshot (literal)"
+msgstr "Screenshot"
```
`msgstr "\"Screenshot\""` is better

amivaleo replied:
Yep. Done.

amivaleo commented:
I updated the ,po file according to your suggestions, and also deleted the old .mo file and uploaded the new one.
I don't know if I have to make another pull request now or if you can already see the changes. I'm still learning how to use github.

OttoAllmendinger commented:
You did it correctly, you can just push new commits to the branch to change the PR.
